### ISSUE TITLE
Accessibility - WCAG 3.1.1.Language of Page, all pages have no language declaration

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#140]
Every web page should have a correct language declaration.
All pages have no language declaration.
For English pages: lang="en"

Original version this was ok
